anubis/src/components/WebSocketDataFeed.vue

```diff
@@ -28,43 +27,58 @@ export default {
   components: { BasicTable, RowLayoutRow, RowLayout },
   data() {
     return {
-      receivedUpdates:
-        [["TrainID", "Station", "old ETA", "Delay", "Cause", "new ETA"],
-        ],
+      receivedUpdates: [],
+      filteredTableData: []
     };
   },
   methods: {
-    connect(stomp, url = "/topic/updates"){
-      this.stompClient = stomp;
+    connect(url = "/topic/updates"){
+      this.stompClient = this.$store.state.websocket;
       this.stompClient.connect(
           {},
-          () => {
+          () =>
             this.stompClient.subscribe(url, update => {
               console.log(update);
-              // this.pushUpdate(update);
-            });
+              this.filter();
+              if (this.receivedUpdates.length < 1){
+                this.pushUpdate(update, true);
+              } else {
+                this.pushUpdate(update, false);
+              }
           },
           error => {
             console.log(error);
-          }
-      );
+          })
+      )
     },
-    pushUpdate(update){
+    pushUpdate(update, header){
```

Comment:
Header seems to be a boolean? We could use typescript types to make all this explicit